deeppavlov/models/torch_bert/torch_transformers_sequence_tagger.py

```diff
@@ -333,10 +333,12 @@ def load(self, fname=None):
 
         if self.pretrained_bert:
             config = AutoConfig.from_pretrained(self.pretrained_bert, num_labels=self.n_classes,
-                                                output_attentions=False, output_hidden_states=False)
+                                                output_attentions=False, output_hidden_states=False, 
+                                                return_dict=False)
             self.model = AutoModelForTokenClassification.from_pretrained(self.pretrained_bert, config=config)
         elif self.bert_config_file and Path(self.bert_config_file).is_file():
             self.bert_config = AutoConfig.from_json_file(str(expand_path(self.bert_config_file)))
+            self.bert_config.return_dict = False
```

Comment:
I suggest not hard-coding this sort of thing. Moreover, using the newer data structure that is returned with return_dict = True doesn't pose much of a problem. You can access relevant elements with dot notation (like `output.loss` etc). Additionally, this appears to be a more robust approach, as opposed to indexing outputs with some magic numbers (like output[0]).